.github/workflows/release.yml

```diff
@@ -14,13 +14,21 @@ jobs:
     runs-on: ubuntu-latest
     environment: BUF
     needs: run_checker
-    if: needs.run_checker.outputs.run_release_proto == 'true' && github.repository_owner == 'astriaorg'
+    if: github.event_name != 'merge_group' && needs.run_checker.outputs.run_release_proto == 'true' && github.repository_owner == 'astriaorg'
```

Comment:
Merge group branch naming is very weird, and offers no real benefit to push to buf here. Whereas pushing PRs enables testing and testing in other repos.